tailwind.config.js

```diff
@@ -6,19 +6,26 @@ export default {
   theme: {
     extend: {
       fontFamily: {
-        sans: ["Montserrat", "system-ui"],
+        sans: ["Inter", "system-ui"],
       },
       colors: {
-        primary: "light-dark(#3f3fcf, #3030c0)",
-        secondary: "light-dark(#a46de3, #531c92)",
-        accent: "light-dark(#12aff3, #0caaed)",
-        "text-base": "light-dark(#212121, #dedede)",
-        "bg-base": "light-dark( #e8e8e8, #171717)",
+        primary:
+          "light-dark(oklch(49.12% 0.3096 285.75), oklch(65.69% 0.196 285.75))",
+        "primary-content":
+          "light-dark(oklch(0.89824 0.06192 285.75), oklch(0.13138 0.0392 285.75))",
+        secondary:
+          "light-dark(oklch(69.71% 0.329 342.55), oklch(74.8% 0.26 342.55))",
+        accent:
+          "light-dark(oklch(76.76% 0.184 183.61), oklch(74.51% 0.167 183.61))",
+        "base-content": "light-dark(#1f2937, #A6ADBB)",
+        "base-100": "light-dark(oklch(100% 0 0), #1d232a)",
+        "base-200": "light-dark(#F2F2F2, #191e24)",
+        "base-300": "light-dark(#E5E6E6, #15191e)",
       },
     },
 
     fontFamily: {
-      keycap: ["OpenGorton", "Montserrat"],
```

Comment:
Is there a particular reason you ditched Gorton? I really enjoy the "classic" look of it, but I'm open to changing it if we really need to.

Reply:
While it is charming, I don't think it fits well for this application. The fact that it has no lowercase characters, very thin characters, and low kerning were factors in my swapping. I also worry about future cases of supporting international characters since Gorton only has about 100 of the most basic glyphs. Maybe in the future we could support alternate themes and fonts in a user setting?